torch/lib/c10d/TCPStore.cpp

```diff
@@ -428,7 +437,7 @@ TCPStore::~TCPStore() {
   if (isServer_) {
     // Store daemon should end because of closed connection.
     // daemon destructor should join the thread
-    tcpStoreDaemon_.reset(nullptr);
+    tcpStoreDaemon_ = nullptr;
```

Comment:
Curious about this change - does it fail if you try to call reset if it is already a nullptr?